biogeophys/FatesHydroWTFMod.F90

```diff
@@ -0,0 +1,1136 @@
+module FatesHydroWTFMod
+
+  use FatesConstantsMod, only : r8 => fates_r8
+  use FatesConstantsMod, only : fates_unset_r8
+  use FatesConstantsMod, only : pa_per_mpa
+  use FatesConstantsMod, only : mpa_per_pa
+  use FatesConstantsMod, only : mm_per_m
+  use FatesConstantsMod, only : m_per_mm
+  use FatesConstantsMod, only : denh2o => dens_fresh_liquid_water
+  use FatesConstantsMod, only : grav_earth
+  use FatesConstantsMod, only : nearzero
+  use FatesConstantsMod, only : pi_const
+  use FatesGlobals     , only : endrun => fates_endrun
+  use FatesGlobals     , only : fates_log
+  use shr_log_mod      , only : errMsg => shr_log_errMsg
+
+  implicit none
+  private
+
+  ! -------------------------------------------------------------------------------------
+  ! This module contains all unit (F)unctions associated with (W)ater (T)ransfer.
+  ! e.g. WTFs
+  ! These are also called "pedotransfer" functions, however, since these
+  ! may be applied to xylems, stems, etc, they are not limited to soils (pedo).
+  ! -------------------------------------------------------------------------------------
+
+  character(len=*), parameter, private :: sourcefile = &
+       __FILE__
+
+
+  real(r8), parameter :: min_ftc = 0.0005_r8   ! Minimum allowed fraction of total conductance
+                                               
+
+  real(r8), parameter :: min_sf_interp = 0.02 ! Linear interpolation above this saturated frac
```

Comment:
The intention here was to prevent flow conditions with exactly zero flow.  This number was intended to be suitably small to avoid the strangeness of zero's in math, but still be small enough to not really have an affect on results.  Note the units here are fraction of conductance, not saturation fraction. This also will not place a lower bound on saturated fraction. I suppose I could just remove this cap.  Thoughts @xuchongang ?

Reply:
I had a similar comment. @rgknox , I think @xuchongang was referring to line 34, not line 31. He meant the comment should read "Linear interpolation below this saturated frac"

I like the min_ftc to avoid zero flow. I agree with your rationale.